pennylane/ops/channel.py

```diff
@@ -365,6 +365,92 @@ def _kraus_matrices(cls, *params):
         return [K0, K1, K2, K3, K4]
 
 
+class PauliError(Channel):
+    r"""PauliError(operators, p, wirelist, wires)
+    Arbitrary number qubit, arbitrary Pauli error channel.
+
+    This class models corresponding Kraus matrices to a series of arbitrary Pauli operators on
+    an arbitrary amount of wires. The operators are supplied in a string. The according wires are
+    supplied in an array called wirelist. The probability with which the channel is applied is
+    supplied as a float. Each wire can take one Pauli operator.
+
+    This channel is modelled by the following Kraus matrices:
+
+    .. math::
+        K_0 = \sqrt{1-p} * I
+
+    .. math::
+        K_1 = \sqrt{p} * (K_{w0} \otimes (K_{w1} \otimes (\dots K_{w_{max}})))
+
+    Where :math:`I` is the Identity,
+    and :math:`\otimes` denotes the Kronecker Product,
+    and :math:`K_wi` denotes the Kraus matrix corresponding to the operator acting on wire i,
+    and :math:`p` denotes the probability with which the channel is applied
+
+    **Warning:**
+    Because the Kraus Matrices scale exponentially to the amount of wires, their dimension blow up
+    fast. Use with caution!
```

Comment:
Using this warning syntax should add a box to the docstring. Also suggested a bit of a rephrasing.
```suggestion
    .. warning::

        The size of the Kraus Matrices for PauliError scale exponentially
        with the number of wires, the channel acts on. Simulations with
        PauliError can result in a significant increase in memory and
        computational usage. Use with caution!
```